lib/em/pure_ruby.rb

```diff
@@ -228,10 +292,47 @@ def set_pending_connect_timeout sig, tm
   end
 end
 
-
 module EventMachine
   # @private
   class Error < Exception; end
+  # @private
+  class ConnectionError < RuntimeError; end
+  # @private
+  class ConnectionNotBound < RuntimeError; end
+  # @private
+  class UnknownTimerFired < RuntimeError; end
+  # @private
+  class Unsupported < RuntimeError; end
+end
```

Comment:
RuntimeError is a child of StandardError, so this is good.

In a separate PR, I am working on getting the rest of EM::Error under StandardError, but RuntimeError may be just as well.